code/flows.rb

```diff
@@ -1,7 +1,10 @@
 Shoes.app do
   flow do
-    para "One"
-    para "Two"
-    para "Three"
+    banner "One"
+    banner "Two"
+    banner "Three"
+    banner "Four"
+    banner "Five"
+    banner "Six"
```

Comment:
I didn't really like the "increase para size" hack, I think using something like banner is more sensible - would remove the hack as well if you agree :)